### ISSUE TITLE
Translation to French

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ejwa/gitinspector#38]
```
This is the .po file for French. I checked the generated .mo gives the expected 
results, except for a few problems with the original .pot file that I'll report 
separately.

Thanks for this great tool btw!
```

Original issue reported on code.google.com by `yannick....@gmail.com` on 2 Nov 2014 at 9:12

Attachments:
* [messages_fr.po](https://storage.googleapis.com/google-code-attachments/gitinspector/issue-38/comment-0/messages_fr.po)

**Comment by adam-waldenberg** [comment]
Hi. A new messages.pot has been added with commit 47b11ad, so an update is needed to the translation file. Refter to the commit diff to see all changes.

Once updated, the translation can be included in the upcoming 0.4.2 release.

**Comment by adam-waldenberg** [comment]
Translation updated with pull request #81 and commits eec6741c and 21c63468.
